djstripe/models.py

```diff
@@ -141,12 +141,15 @@ def _attach_objects_hook(self, cls, data):
 
 @class_doc_inherit
 class Customer(StripeCustomer):
-    doc = """
+    """
+    A model extending StripeCustomer providing dj-stripe functionality.
 
-.. note:: Sources and Subscriptions are attached via a ForeignKey on StripeSource and Subscription, respectively. \
-Use ``Customer.sources`` and ``Customer.subscriptions`` to access them.
+    .. note:: Sources and Subscriptions are attached via a ForeignKey on StripeSource and Subscription, respectively.
+    Use ``Customer.sources`` and ``Customer.subscriptions`` to access them.
     """
-    __doc__ = getattr(StripeCustomer, "__doc__") + doc
+
+    # append docstring to super class docstring
+    __doc__ += getattr(StripeCustomer, "__doc__")
```

Comment:
Hmmm, I see, I see. What should be the desired behaviour then? Not having this line at all so the `Customer` objects docs are shown alone in the docs?